src/components/PolicyMasterPanel.js

```diff
@@ -362,6 +370,62 @@ class PolicyMasterPanel extends FormPanel {
                   onChange={(v) => this.updateAttribute("status", v)}
                 />
               </Grid>
+              <Grid xs={12}>
+                <Grid item xs={3} className={classes.item}>
+                  <FormControlLabel
+                    control={
+                      <Checkbox
+                        color="primary"
+                        checked={edited?.isPaid}
+                        onChange={(e) =>
+                          this.updateAttribute("isPaid", e.target.checked)
+                        }
+                      />
+                    }
+                    disabled={readOnly}
+                    label={formatMessage(
+                      intl,
+                      "policy",
+                      "Policy.payInOneInstallment"
+                    )}
+                  />
+                </Grid>
+              </Grid>
+              {edited?.isPaid && (
```

Comment:
To be confirmed, but I think this option should be available only during creation of a policy, and not during editing. So this should be changed accordingly.

Additionally - isPaid is only defined as GQL variable and not in model. So isPaid will always be false/null. Is Paid is parameter for mutation to create the policy and not stored afterwards.

Reply:
After changes, "Pay in one installment" option is visible only during creation of policy.